techbuzz/src/main/java/com/sivalabs/techbuzz/posts/web/controllers/ViewCategoryController.java

```diff
@@ -32,9 +32,11 @@ public String viewCategory(
             @PathVariable(name = "categorySlug") String categorySlug,
             @RequestParam(name = "page", defaultValue = "1") Integer page,
             Model model) {
-        log.info("Fetching posts for category {} with page: {}", categorySlug, page);
+        log.info("GET /c/{} - Fetching posts for category {} with page: {}", categorySlug, categorySlug, page);
         PagedResult<PostViewDTO> data = postService.getPostsByCategorySlug(categorySlug, page);
         if (data.data().isEmpty() && (page > 1 && page > data.totalPages())) {
+            log.warn("No posts found for category: {}", categorySlug);
```

Comment:
These 2 log statements can be combined into a single one:

log.warn("No posts found for category: {}, page:{}. Redirecting to last page", categorySlug, page);